dot/types/inherents.go

```diff
@@ -63,7 +63,7 @@ func (d *InherentsData) SetInt64Inherent(key []byte, data uint64) error {
 	val := make([]byte, 8)
 	binary.LittleEndian.PutUint64(val, data)
 
-	venc, err := scale.Encode(val)
+	venc, err := scale.Marshal(val)
```

Comment:
with the new SCALE code, I guess is just `scale.Marshal(data)` without that `LittleEndian.PutUint64` call, right?

Reply:
Im not sure if I can make this change because the map data is a []byte and if I encode the uint64 as is it will be different than the []byte encoding

Reply:
hm, I told that because I used the `scale.Marshal(uint32(len(data)))` and the runtime method does not complain about the bytes format

Reply:
I'm not entirely sure what you mean I don't think. My concern is when you encode a slice the leading byte represents the length of the slice, followed by the data and encoding a uint64 is just the data. So I think in this case the encoding would be different if I just directly encoded the uint64